config/PropertyNames.xml

```diff
@@ -409,6 +409,11 @@
         <property name="Node.Server" class="ObjectAdapter" languages="cpp" />
         <property name="Node.Server.Enabled" default="1" languages="cpp" />
         <property name="Node.Server.ForwardDiscoveryToMulticast" default="0" languages="cpp" />
+        <property name="Topic.ClearHistory" default="OnAll" languages="cpp" />
+        <property name="Topic.DiscardPolicy" default="Never" languages="cpp" />
+        <property name="Topic.Priority" default="0" languages="cpp" />
+        <property name="Topic.SampleCount" default="-1" languages="cpp" />
+        <property name="Topic.SampleLifetime" default="0" languages="cpp" />
```

Comment:
Topic properties where not added before, but they are public and documented in

https://doc.zeroc.com/datastorm/latest/property-reference/datastorm-topic

I updated the docs to include `-1` default for SampleCount this was missing in the docs.